### PR TITLE
#MAG-391 fix several style issues

### PR DESCRIPTION
**Pull request by ThomasSevagen**
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/board-list/utils.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Board } from "~/models/board.model";
+
+export const isBoardInFilter = (board: Board, searchText: string) => {
+  if (
+    board.title.toLowerCase().includes(searchText.toLowerCase()) ||
+    (board.description != null &&
```
Comment (niko-amosse) [inline frontend/src/components/board-list/utils.ts:6]: c'est hooooonteux

File: frontend/src/components/board-list/utils.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      board.description.toLowerCase().includes(searchText.toLowerCase()))
+  )
+    return true;
+  if (board.tags == null || board.tags.length == 0) {
```
Comment (niko-amosse) [inline frontend/src/components/board-list/utils.ts:10]: HOOOONTEEEEEUUUUX

File: frontend/src/components/folder-item/FolderItem.tsx [hunk 3]
```diff
@@ -186,6 +186,7 @@ export const FolderItem: React.FunctionComponent<FolderListProps> = ({
   const resetDragAndDrop = (): void => {
     onDragAndDrop(undefined);
   };
+  console.log(folder);
```
Comment (niko-amosse) [inline frontend/src/components/folder-item/FolderItem.tsx:189]: HOOOOOONTEEUUUUUUUUUUUUUUX

**Review by niko-amosse — APPROVED** [review]
HONTEUX CETTE PR

**Review by Mattox0 — APPROVED** [review]

**Pull request merged**